data_filter_elasticsearch/internal/opa/opa.go

```diff
@@ -71,9 +71,9 @@ func Compile(ctx context.Context, input map[string]interface{}, policy []byte) (
 
 func processQuery(pq *rego.PartialQueries) (Result, error) {
 
-	queries := []elastic.Query{}
+	queries := make([]esquery.Mappable, 0, 100)
```

Comment:
Maybe it's obvious, but what does the size/length mean here? Maybe use a named var.

Reply:
Whoops didn't manage to fix it before you merged 😅

Reply:
Heh, it was such a minor remark that I figured I'd go ahead anyway :) Awesome contribution 👍